core/src/main/java/org/apache/brooklyn/feed/ssh/SshFeed.java

```diff
@@ -270,6 +270,7 @@ private SshPollValue exec(String command, Map<String,String> env) throws IOExcep
         ConfigBag flags = ConfigBag.newInstanceExtending(config().getBag())
             .configure(SshTool.PROP_NO_EXTRA_OUTPUT, true)
             .configure(SshTool.PROP_OUT_STREAM, stdout)
+            .configure(SshMachineLocation.CLOSE_CONNECTION, true)
```

Comment:
This is really expensive. It means that we close all connections in our ssh-pool-cache in `SshMachineLocation`.
(As an aside, I wonder if that works properly! I hope that invalidating the cache won't terminate other ssh commands that are executing in other threads!).